inline.js

```diff
@@ -184,6 +193,31 @@ function rehydrateAttributes(attributes) {
           `Inlined a map, pushed ${visToInline.references.length} inner references`
         );
         counter.map.add(ref.id);
+      } else if (ref && migratedSearch.has(ref.id)) {
+        const searchToInline = migratedSearch.get(ref.id);
+        p.version = searchToInline.migrationVersion.search; // TODO - check this
+        p.type = "search";
+        p.title = searchToInline.title;
+        p.description = searchToInline.description;
+        p.embeddableConfig.attributes = {
+          sort: searchToInline.attributes.sort,
+          columns: searchToInline.attributes.columns,
+          kibanaSavedObjectMeta:
+            searchToInline.attributes.kibanaSavedObjectMeta,
+          references: searchToInline.references,
+        };
```

Comment:
@elastic/kibana-data-discovery this is the key transformation I want your validation on.